test_fms/mpp/test_mpp_domains.bats

```diff
@@ -131,30 +131,30 @@ teardown () {
     then
       skip "Does not work on Darwin"
     fi
-    sed "s/test_interface = .false./test_interface = .true./" input.nml_base > input.nml
+    sed "s/test_interface = .false./test_interface = .true./" $srcdir/test_fms/mpp/input_base.nml > input.nml
     run mpirun -n 2 ./test_mpp_domains
     [ "$status" -eq 0 ]
 }
 
-@test "14: Test Check Parallel" {
-    if [ "$skip_test" = "true" ]
-    then
-      skip "Does not work on Darwin"
-    elif [ "x$TRAVIS" = "xtrue" ]
-    then
-      skip "Fails on Travis"
-    fi
-    sed "s/check_parallel = .false./check_parallel = .true./" input.nml_base > input.nml
-    run mpirun -n 2 ./test_mpp_domains
-    [ "$status" -eq 0 ]
-}
+# @test "14: Test Check Parallel" {
```

Comment:
This appears to be based on #200.  Please update #200 to `skip` this test, and then ensure this file is updated here as well.